### ISSUE TITLE
multilanguage links generated wrong / wrong korean link snipplet

### ISSUE THREAD

**Issue by icq4ever** [openframeworks/ofSite#648]
### links lead to wrong address 

for example : 
https://openframeworks.cc > korean link -> https://openframeworks.cc/ja//
https://openframeworks.cc/download > japanese link -> https://openframeworks.cc/ja//download

### also, korean link name generated to "영어", which means "english". 
it need to be "한국어", which means "korean"

this happen already before. I already fix on [transifex](https://www.transifex.com/ralsina/nikola/viewstrings/#ko/messages). also, asked to [nikola](https://www.transifex.com/ralsina/nikola/viewstrings/#ko/messages). and they said it already patched. maybe related with nikola's version ?

**Comment by arturoc** [comment]
yeah it's surely because of that, i updated everything to latest versions yesterday in the server and that seems to have broken this let me check

**Comment by arturoc** [comment]
i've gone back to a backup of the old install and it seems to have fixed it, if you can check that it's correct now? i've also asked in your issue in the nikola github to see if we can somehow update nikola without this breaking

**Comment by icq4ever** [comment]
seems everything's ok. :) hope to solve that issue with nikola.
